pkg/types/trade.go

```diff
@@ -193,6 +194,7 @@ func (trade Trade) String() string {
 		trade.ID,
 		trade.Time.Time().Format(time.StampMilli),
 		trade.Liquidity(),
+		trade.OrderUUID,
```

Comment:
UUID is optional, I think you need to add a check or the text will be too long

Reply:
ok